dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -0,0 +1,73 @@
+class LtiV1Controller < ApplicationController
+  # Don't require an authenticity token because LTI Platforms POST to this controller.
+  skip_before_action :verify_authenticity_token
+
+  # [GET/POST] /lti/v1/login(/:platform_id)
+  def login
+    if params[:client_id]
+      query_params = {client_id: params[:client_id], issuer: params[:iss]}
+    elsif params[:platform_id]
+      query_params = {platform_id: params[:platform_id]}
+    else
+      return unauthorized_status
+    end
+
+    begin
+      lti_integration = LtiIntegration.find_by!(query_params)
+    rescue ActiveRecord::RecordNotFound
+      return unauthorized_status
+    end
+
+    auth_request_body = {
+      scope: "openid",
+      response_type: "id_token",
+      client_id: lti_integration[:client_id],
+      redirect_uri: "https://studio.code.org/lti/v1/authenticate", # TODO: where can we configure this to be dynamic depending on deployment environment, i.e. prod, localhost, etc?
```

Comment:
Found the method I was looking for:
```
redirect_uri: CDO.studio_url('/lti/v1/authenticate', CDO.default_scheme),
```

Reply:
Does the `/lti/v1/authenticate` endpoint exist?

Reply:
Not yet. That's the next Jira item we'll work on.